src/js/tracing/reactnavigation.ts

```diff
@@ -188,8 +222,51 @@ export class ReactNavigationInstrumentation extends InternalRoutingInstrumentati
     if (route) {
       if (this._latestTransaction) {
         if (!previousRoute || previousRoute.key !== route.key) {
-          const originalContext = this._latestTransaction.toContext() as typeof BLANK_TRANSACTION_CONTEXT;
           const routeHasBeenSeen = this._recentRouteKeys.includes(route.key);
+          const latestTtidSpan =
+            !routeHasBeenSeen &&
+            this._options.enableTimeToInitialDisplay &&
+            startInactiveSpan({
+              op: 'ui.load.initial_display',
+              name: `${route.name} initial display`,
+              startTimestamp: this._latestTransaction?.startTimestamp,
+            });
+
+          !routeHasBeenSeen &&
+            this._newScreenFrameEventEmitter?.once(
+              NewFrameEventName,
+              ({ newFrameTimestampInSeconds }: NewFrameEvent) => {
+                if (!latestTtidSpan) {
+                  return;
+                }
+
+                if (spanToJSON(latestTtidSpan).parent_span_id !== getActiveSpan()?.spanContext().spanId) {
```

Comment:
why are you using `spanToJSON(latestTtidSpan).parent_span_id` instead of `latestTtidSpan.parentSpanId`?

Reply:
Because `latestTtidSpan.parentSpanId` will be removed in JS SDK v8.